crates/vm/levm/src/precompiles.rs

```diff
@@ -421,27 +428,20 @@ pub fn ecadd(
         .concat();
         Ok(Bytes::from(res))
     } else if second_point_is_zero {
-        // If second point is zero, return is first point
+        // If second point is zero, response is first point
         let first_point = BN254Curve::create_point_from_affine(first_point_x, first_point_y)
             .map_err(|_| PrecompileError::ParsingInputError)?;
         let res = [first_point.x().to_bytes_be(), first_point.y().to_bytes_be()].concat();
         Ok(Bytes::from(res))
     } else {
-        // If none of the points is zero, return is the sum of both in the EC
+        // If none of the points is zero, response is the sum of both in the EC
```

Comment:
ditto

Reply:
Done in [Use main's version in ecmul and ecadd](https://github.com/lambdaclass/ethrex/pull/1560/commits/cd4a769adbcfd1d214ddbf905053e1ff3d74408d)